README.md

```diff
@@ -47,6 +47,12 @@ g.AddTsigToDomain("example.com", tsig.UUID)
 for _, host := range []string{"198.51.100.1", "2001:DB8::1"} {
     g.AddSlaveToDomain("example.com", host)
 }
+// Step 8: Create a TSIG key
```

Comment:
There are no steps 8, 9 nor 10 on http://doc.livedns.gandi.net/#quick-example
Please stick to the official gandi example : the steps here simply mimic the official documentation.